DebugLibrary.py

```diff
@@ -564,6 +567,20 @@ def complete_keywords(self, text, line, begin_idx, end_idx):
 
     complete_k = complete_keywords
 
+    def do_docs(self, kw_name):
+        """Get keyword documentation for individual keywords
+
+         d(ocs) [<keyword_name>]
+        """
+
+        for lib in get_libs():
+            for keyword in get_lib_keywords(lib, True):
```

Comment:
Named arguments are meant to increase readability: `get_lib_keywords(lib, long_format=True)`

Reply:
True. Fixed with commit below